bin/uploadTemplates.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
+const path = require('path');
+const env = require('../env.json');
+const mandrill = require('mandrill-api/mandrill');
+const mandrill_client = new mandrill.Mandrill(env.MANDRILL_API_KEY);
+
+const fetchTemplates = async () => {
+  return new Promise((resolve, reject) => {
+    mandrill_client.templates.list(null, 
+      result => resolve(result), 
+      error => reject(error)
+    );
+  });
+};
+
+const getLocalTemplates = async () => {
+  return new Promise((resolve, reject) => {
+    fs.readdir('./email/', (error, files) => {
+      if (error) {
+        return reject(error);
+      }
+      return resolve(files);
+    });
+  });
+};
+
+const getSlugs = templates => {
+  return templates.map(template => {
+    const [slug, extension] = template.split('.');
+    return slug;
+  });
+};
+
+const uploadTemplates = templates => {
+  const promises = templates.map(template => new Promise((resolve, reject) => {
+    fs.readFile(`./email/${template.slug}.hbs`, 'utf8', function(err, contents) {
+      if (err) {
+        reject(err);
```

Comment:
I think you need to return here.

Or, if you use the promisify tweak above, you shouldn't need to worry about it since the error should get thrown for you.

```suggestion
        return reject(err);
```